cypress/e2e/workspace.spec.js

```diff
@@ -84,17 +84,19 @@ describe('Workspace', function() {
 			.should('be.visible')
 	})
 
-	it('formats text', function() {
+	it('formats text', function() {		
 		cy.visit(`apps/files?dir=/${encodeURIComponent(currentFolder)}`)
-		cy.openWorkspace().type('Format me')
-		cy.getContent().type('{selectall}')
-		;[
+		cy.openWorkspace()
+
+		;const buttons = [
```

Comment:
The extra `;` at the beginning of the line was here because the line started with an array:
```js
cy.getContent().type('{selectall}')
;[
```
Without a `;` Javascript  would interpret the `[` as an index into an array (as in  `cy.getContent().type('{selectall}')[1]`)

Now that you are using a variable (much better) there's no more need for the semicolon.